pkg/authorization/api/validation/validation.go

```diff
@@ -103,6 +103,13 @@ func ValidateRole(role *authorizationapi.Role) fielderrors.ValidationErrorList {
 	return allErrs
 }
 
+func ValidateRoleUpdate(oldRole *authorizationapi.Role, role *authorizationapi.Role) fielderrors.ValidationErrorList {
```

Comment:
flip the ordering to be role, oldRole so its consistent with the strategy interface.